web/src/pages/CustomerSatisfactionPage/CustomerSatisfactionPage.tsx

```diff
@@ -14,8 +14,10 @@ const CustomerSatisfactionPage = () => {
   const { currentUser } = useAuth()
   const [step, setStep] = useState(1)
   const handleNextStep = () => {
+    if (step == 4) {
+      navigate(routes.endSurvey(), {replace: true})
```

Comment:
I would add a return here, so you do not inadvertently increase the step.

Reply:
Yes!